apiary.apib

```diff
@@ -466,33 +480,34 @@ A common question is whether or not to include multiple recipients on a single P
     + Attributes
         + info
             + code (string) - Status code for request. Possibilities:
-                    `INFO_CAMPAIGN_CREATED` - Campaign creation has begun successfully
-                    `INFO_CAMPAIGN_CREATED_FUNDS_REQUIRED` - Campaign creation request was successfully received by Giftbit but there are insufficient funds in your Giftbit account to cover the cost of the campaign. Note you do not need to resend the campaign. It will automatically be sent when there are enough funds in your account.
-                    `INFO_CAMPAIGN_CREATED_FUNDS_PENDING` - The campaign creation request was successfully received by Giftbit but pending funds (such as a cheque or wire transfer that you sent) have not yet cleared. Note you do NOT need to resend the campaign as soon as your pending funds clear and become available, the campaign will be automatically delivered to your recipients.
+                    `INFO_CAMPAIGN_CREATED` - Order creation has begun successfully
+                    `INFO_CAMPAIGN_CREATED_FUNDS_REQUIRED` - Order creation request was successfully received by Giftbit but there are insufficient funds in your Giftbit account to cover the cost of the order. Note you do not need to resend the order. It will automatically be sent when there are enough funds in your account.
+                    `INFO_CAMPAIGN_CREATED_FUNDS_PENDING` - The order creation request was successfully received by Giftbit but pending funds (such as a cheque or wire transfer that you sent) have not yet cleared. Note you do NOT need to resend the order as soon as your pending funds clear and become available, the order will be automatically delivered to your recipients.
 
         + campaign
-            + message (string) - The message for the campaign.
-            + subject (string) - The subject for the campaign.
-            + contacts (object) - A list of contacts the campaign is being sent to.
-            + price_in_cents (number) - The amount of the gift offer to each recipient.
-            + brand_codes (array[string]) - The brand(s) of gift cards sent.
-            + status (string) - The status of the campaign. Possibilities:
-                API_CREATING - The campaign is in the process of being created.
-                AWAITING_FUNDS - The campaign has not yet been delivered to the recipient(s)
+            + message (string) - The message for the order.
+            + subject (string) - The subject for the order.
+            + contacts (object) - A list of contacts the order is being sent to.
+            + price_in_cents (number) - The amount of the reward offer to each recipient.
+            + brand_codes (array[string]) - Custom list of reward brand(s) sent.
+            + region (string) - Full Catalog region name. Possibilities: United States, Canada
```

Comment:
I'm not sure it makes sense to list a specific set of possibilities here since we expect supported regions are expected to grow soon. Also, the region name we have for the United States is "USA"

Reply:
We want to list the options somewhere I think, as they can type one we don't have and be confused when it doesn't work. In my pov I think we might need to update this whenever we add new options.